Odata-docs/webapi/model-builder-capabilities-vocabulary.md

```diff
@@ -0,0 +1,251 @@
+---
+title: " Non-convention model builder with capabilities vocabulary"
+description: "capabilities vocabulary"
+author: g2mula
+ms.author: g2mula
+ms.date: 8/17/2020
+---
+# Non-convention model builder with capabilities vocabulary
+**Applies To**:[!INCLUDE[appliesto-webapi](../includes/appliesto-webapi-v7.md)][!INCLUDE[appliesto-webapi](../includes/appliesto-webapi-v6.md)]
+
+The model builder now allows you to configure the [capabilities of a service](https://github.com/oasis-tcs/odata-vocabularies/blob/master/vocabularies/Org.OData.Capabilities.V1.md). Please follow the link for more information on capabilities vocabulary.
+
+### Examples
+Find below some sample usage...
+
+Unless otherwise specified, examples will be based on the model below.
+```csharp
+public class Customer
+{
+  public int CustomerId { get; set; }
+  public string Name { get; set; }
+}
+
+var customerConfiguration = builder.EntitySet<Customer>("Customers");
```

Comment:
Maybe also show where `builder` comes from in this code snippet. like a `var builder = ....` statement.